### DISCUSSION TITLE
Using iPhone as a hotspot instead of LTE modem?

### DISCUSSION THREAD

**Discussion by G0lexiy** [tesla-android/issue-tracker#212] [Q&A]
Is it possible to use my iPhone as a hotspot instead of buying lte modem? I can't seem to find instructions how to do it. I connected the car to the rpi using Tesla android Wi-Fi, and connected rpi to my iPhone using Bluetooth. Anything else I need to do to make it work?

**Comment by ks0002544** [comment]
In your schema Pi4 is hotspot. Tesla connects to Pi4 hotspot and you can use 9.9.0.1 in browser Tesla to connect Android Auto (Pi4). After as I understand you can use usb modem for internet connection or use your smartphone as modem. I also tried to use bluetooth but no chance. Only wired connection between Pi4 and my smartphone works. But usb socket power is not enough to power such more units. Next time I plan to check 12v-usb adapter.
